src/pages/AdminConsole.js

```diff
@@ -177,29 +162,9 @@ class AdminConsole extends Component {
                   unreviewedCases: dulUnreviewedCases
                 }),
               ]),
-
-              div({ className: "col-lg-6 col-md-6 col-sm-12 col-xs-12 admin-box" }, [
-
-                AdminConsoleBox({
-                  id: 'btn_addDUL',
-                  clickHandler: this.addDul,
-                  color: 'dul',
-                  title: 'Add Data Use Limitations',
-                  description: 'Catalog a Data Use Limitation Record in the system',
-                  iconName: 'add-dul',
-                  iconSize: 'default',
-                }),
-                AddDulModal({
-                  isRendered: this.state.showAddDulModal,
-                  showModal: this.state.showAddDulModal,
-                  editMode: false,
-                  onOKRequest: this.okModal,
-                  onCloseRequest: this.closeModal,
-                  onAfterOpen: this.afterModalOpen,
-                  dul: '',
-                  editConsent: ''
-                }),
-              ]),
+              div({
+                style: { margin: "10px", padding: "0px", display: "inline-block" },
+                className: "col-lg-6 col-md-6 col-sm-12 col-xs-12" }, [])
```

Comment:
This is the empty placeholder block that allows for the button to its right to flex correctly as the page is resized.